third_party/BUILD.bazel

```diff
@@ -0,0 +1,20 @@
+load("@rules_foreign_cc//foreign_cc:defs.bzl", "configure_make")
+
+filegroup(
+    name = "ghostscript_dependencies",
+    srcs = [
+        "@texlive_texmf__texmf-dist__dvips__base",
+    ],
+    visibility = ["//visibility:public"],
+)
+
+configure_make(
+    name = "lib_ghost_script_configure",
+    configure_in_place = True,
+    autogen = True,
+    install_prefix = "output",
+    lib_source = "@ghost_script_source//:all_srcs",
+    targets = ["so", "soinstall"],
```

Comment:
Similar as above, I suspect this isn't compatible with other platforms than Linux?

Reply:
You are right. Let me see if I can manage to fix that.

Reply:
"so" and "soinstall" as far as I've understood is only the generic make commands, the host platform will affect output artifact as I understood it